trinity/nodes/base.py

```diff
@@ -82,3 +114,23 @@ def start_loop(loop):
         thread.start()
 
         return new_loop
+
+
+def create_db_manager(ipc_path: str) -> BaseManager:
```

Comment:
We should use a `PurePath` here and downcast to a `str` before passing into the `DBManager`